### PR TITLE
set validatorUrl to null (#323)

### PR DESCRIPTION
**Pull request by MaxKramnik**

### CODE REVIEW

**Comment by whitlockjc** [comment]
Do you mind amending your commit to update the [release notes](https://github.com/apigee-127/swagger-tools/blob/master/RELEASE_NOTES.md#tbd) for this change, and update the commit to mention that it fixes #370 as well.  I'll merge promptly after.

**Comment by MaxKramnik** [comment]
Done. Thank you!

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 * Fix issue where the `failedValidation` property was not set for `Content-Type` request validation errors _(PR #420)_
 * Fix issue where `array` `body` parameters would coerce non-array types into an array _(Issue #438)
 
+### 0.10.2 (2017-05-24)
```
Comment (whitlockjc) [inline RELEASE_NOTES.md]: Put this under the `TBD` section please.

**Comment by MaxKramnik** [comment]
Done.

**Comment by whitlockjc** [comment]
Thanks a lot!

**Comment by MaxKramnik** [comment]
Any plans to bump up a version soon?

**Comment by whitlockjc** [comment]
I am working to get `swagger-tools` and sway bumped to `json-refs@v3.0.0`.  During this effort, bugs will be fixed and a release will be created.  I'm just not sure of an ETA.

**Pull request merged**